code/lib/types/src/modules/csf.ts

```diff
@@ -20,7 +124,7 @@ export type {
   Globals,
   GlobalTypes,
   IncludeExcludeOptions,
-  InputType,
+  InputTypeOverride as InputType,
```

Comment:
Don't export from here and require the mess of imports above. Just remove the original export and export it separately (see next comments).

```suggestion
```